.grit/patterns/go/hidden_goroutine.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Detected a hidden goroutine
+---
+
+Detected a hidden goroutine. Function invocations are expected to synchronous, and this function will execute asynchronously because all it does is call a goroutine. Instead, remove the internal goroutine and call the function using `go`
+
+tags: #correctness #best-practice
+
+```grit
+language go
+
+`func $func() { 
+    go func() {
+        $funcBody 
+    }() 
+}` => `func $func() { 
+    $funcBody 
+}`
```

Comment:
We should probably at least look for *calls* in the same program and add go to them, since removing the go here is only half the fix.